lightning/src/routing/scoring.rs

```diff
@@ -642,15 +642,15 @@ pub struct ProbabilisticScoringFeeParameters {
 impl Default for ProbabilisticScoringFeeParameters {
 	fn default() -> Self {
 		Self {
-			base_penalty_msat: 500,
-			base_penalty_amount_multiplier_msat: 8192,
-			liquidity_penalty_multiplier_msat: 30_000,
-			liquidity_penalty_amount_multiplier_msat: 192,
+			base_penalty_msat: 1000,
+			base_penalty_amount_multiplier_msat: 1_000_000,
```

Comment:
That's a bit hefty `base_penalty_amount_multiplier_msat`, isn't it? :)
https://github.com/MutinyWallet/mutiny-node/blob/68d545d1f82ba0d0e64980d056dc16ee7f00d457/mutiny-core/src/node.rs#L1944
```suggestion
			base_penalty_amount_multiplier_msat: 8192 * 100,
```